### DISCUSSION TITLE
Class accept inheritance, but is disabled

### DISCUSSION THREAD

**Discussion by designjpp** [pimcore/pimcore#17230] [Q&A / Support] [answered]
Hello 

I tested in the Official Demo. The BodyStyle class supports inheritance, but is disabled

![image](https://github.com/pimcore/pimcore/assets/60102400/2403a2d1-6748-4f53-8e0f-acadc06b0974)

![image](https://github.com/pimcore/pimcore/assets/60102400/5d464f8e-41fd-4720-93a8-5f95947b20fd)

Is this a bug?

**Comment by fashxp** [comment, marked as answer]
Nope, it is always possible to place data objects as children from other data objects. 
`Allow inheritance` just activates data inheritance, meaning that children object will inherit value from their parents.
